src/user/types/userResponce.interface.ts

```diff
@@ -0,0 +1,6 @@
+// import { UserType } from "./user.types";
+
+
+// export interface UserResponseInterface {
+//   buyer: UserType & { token: string };
+// }
```

Comment:
Remove the commented code or the entire file

Reply:
✅